projects/js-packages/idc/components/idc-screen/screen-main.jsx

```diff
@@ -38,25 +42,56 @@ const ScreenMain = props => {
 			</h2>
 
 			<p>
-				{ createInterpolateElement(
-					customContent.mainBodyText ||
-						__(
-							'Your site is in Safe Mode because you have 2 Jetpack-powered sites that appear to be duplicates. ' +
-								'2 sites that are telling Jetpack they’re the same site. <safeModeLink>Learn more about safe mode.</safeModeLink>',
-							'jetpack'
-						),
-					{
-						safeModeLink: (
-							<a
-								href={ customContent.supportURL || getRedirectUrl( 'jetpack-support-safe-mode' ) }
-								rel="noopener noreferrer"
-								target="_blank"
-							/>
-						),
-						em: <em />,
-						strong: <strong />,
-					}
-				) }
+				{ ! isDevelopmentSite
+					? createInterpolateElement(
+							customContent.mainBodyText ||
+								__(
+									'Your site is in Safe Mode because you have 2 Jetpack-powered sites that appear to be duplicates. ' +
+										'2 sites that are telling Jetpack they’re the same site. <safeModeLink>Learn more about safe mode.</safeModeLink>',
+									'jetpack'
+								),
+							{
+								safeModeLink: (
+									<a
+										href={
+											customContent.supportURL || getRedirectUrl( 'jetpack-support-safe-mode' )
+										}
+										rel="noopener noreferrer"
+										target="_blank"
+									/>
+								),
+								em: <em />,
+								strong: <strong />,
+							}
+					  )
+					: createInterpolateElement(
+							customContent.mainBodyText ||
+								sprintf(
+									/* translators: %1$s: The current site domain name. %2$s: The original site domain name. */
+									__(
+										'<span>Your site is in Safe Mode because <hostname>%1$s</hostname>, appears to be a staging or development copy of <hostname>%2$s</hostname>.</span>' +
```

Comment:
```suggestion
										'<span>Your site is in Safe Mode because <hostname>%1$s</hostname> appears to be a staging or development copy of <hostname>%2$s</hostname>.</span>' +
```